pyrsa/util/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""
+utility functions
+"""
+
+# from pyrsa.util.indicator import identity, allpairs, identity_pos
```

Comment:
perhaps remove commented-out code before merging to master

Reply:
Thanks for the note! Will clean up and push next week.

Reply:
fixed and pushed.